examples/05_deduplication.py

```diff
@@ -74,27 +75,28 @@
 plt.ylabel("Counts")
 
 ###############################################################################
-# We can now see clearly the structure of the data: The three original medications
-# are the most common ones, however there are many spelling mistakes and hence
-# many slight variations of the names of the original medications.
+# We can now see clearly the structure of the data:
+# the three original medications are the most common ones,
+# but there are many spelling mistakes or
+# slight variations of the original names.
 #
-# The idea is to use the fact that the string-distance of each misspelled medication
-# name will be closest to either the correctly or incorrectly spelled orginal
+# The idea is to use the fact that the string distance of misspelled
+# medications will be closest to their original (most frequent)
 # medication name - and therefore form clusters.
 
 ###############################################################################
-# We can visualize the pair-wise distance between all medication names
-# --------------------------------------------------------------------
+# Visualizing the pair-wise distance between all names
+# ----------------------------------------------------
```

Comment:
Can we move this section below:
- First show the user how to use the deduplicate function, as this is what the busy user is focused on
- Second explain to the user how this works